clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -635,6 +639,9 @@ public void handle(SyncGroupResponse syncResponse,
                 } else if (error == Errors.REBALANCE_IN_PROGRESS) {
                     log.debug("SyncGroup failed because the group began another rebalance");
                     future.raise(error);
+                } else if (error == Errors.FENCED_INSTANCE_ID) {
+                    log.error(fencedInstanceErrorMessage, groupInstanceId, this.getClass().getName());
```

Comment:
We shouldn't need to include the classname here. The `LogContext` should already tell us whether this was a consumer or something else. In fact, the group instance Id should also be added to the log context when constructing the consumer. That will ensure it gets included in every log message from the consumer, which makes it easier to follow logs which include multiples consumers.